src/Mcfedr/AwsPushBundle/Service/Messages.php

```diff
@@ -110,6 +110,9 @@ public function send($message, string $endpointArn)
                 'TargetArn' => $endpointArn,
                 'Message' => $this->encodeMessage($message),
                 'MessageStructure' => 'json',
+                'MessageAttributes' => [
+                    'AWS.SNS.MOBILE.APNS.PUSH_TYPE' => ['DataType' => 'String', 'StringValue' => $message->isContentAvailable() ? Message::PUSH_TYPE_BACKGROUND : $message->getPushType()],
```

Comment:
This is confusing, I think here id just have getpushtype - maybe when set content available set the type to background, and have the default alert